CleanWeather/Models/Coordinates.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Coordinates.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 20/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct Coordinates {
+
+    var lat: Double
+    var lon: Double
+
+    func coordinatesString() -> String {
```

Comment:
let coordinates = Coordinates()
coordinates.coordinatesString() - looks bad

rename this method to toString() 
and even better to have computed property:
var stringValue: String { return ... }